### PR TITLE
Clears the resourceURIMessage when clicking back into the editor screen

### LINKED ISSUE #1014: ResourceURIMessage persists after loading new Resource Template
Creating an publishing an ISSN displays the expected `ResourceURIMessage` as seen here:
Screen Shot 2019-07-19 at 11 06 28 AM

Loading a different resource template still have that message persisting:
Screen Shot 2019-07-19 at 11 09 10 AM

### PR DESCRIPTION
**Pull request by aaron-collier**
Fixes #1014

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -6560,28 +6560,27 @@
       "dependencies": {
         "abbrev": {
           "version": "1.1.1",
-          "resolved": false,
+          "resolved": "",
```
Comment (justinlittman) [inline package-lock.json]: Why are there changes to package-lock.json?

File: src/actionCreators/messages.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+// Copyright 2019 Stanford University see LICENSE for license
+
+import { clearingResourceURIMessage } from 'actions/index'
+
+const clearResourceURIMessage = () => (dispatch, getState) => {
```
Comment (justinlittman) [inline src/actionCreators/messages.js]: Why is there an actionCreator? The action isn't asynchronous.

File: package-lock.json (hunk 1 above)
Comment (aaron-collier) [reply package-lock.json]: This happens ALL THE TIME... I accidentally checked it in... ugh...

File: src/actionCreators/messages.js (hunk 2 above)
Comment (aaron-collier) [reply src/actionCreators/messages.js]: I think I can simplify this into just the template. I was following the pattern used in `loadLanguages` but it seems unnecessary.

File: package-lock.json (hunk 1 above)
Comment (aaron-collier) [reply package-lock.json]: Removed these changes.

File: src/reducers/index.js [hunk 3]
```diff
@@ -3,8 +3,8 @@
 import { combineReducers } from 'redux'
 import authenticate from './authenticate'
 import {
-  removeMyItem, setItemsOrSelections, setBaseURL, showResourceURIMessage, setMyItemsLang,
-  showGroupChooser, closeGroupChooser, showRdfPreview,
+  removeMyItem, setItemsOrSelections, setBaseURL, showResourceURIMessage, clearResourceURIMessage,
+  setMyItemsLang, showGroupChooser, closeGroupChooser, showRdfPreview,
```
Comment (ndushay) [inline src/reducers/index.js]: for some reason, "setMyItemsLang" makes me think of "how's my driving?"

**Pull request merged**